src/app/Components/ArtworkFilter/ArtworkFilterHelpers.ts

```diff
@@ -584,3 +615,27 @@ export const getFilterArrayFromQueryParams = (queryParams: {
     })
   )
 }
+
+export const allowedFilterParams = (routeParams: {
+  [key: string]: string | number | boolean | string[]
+}) => {
+  return pick(routeParams, Object.keys(QueryParamsToFilterValueMapping))
+}
+
+export const getFilterParamsFromRouteParams = (routeParams: {
+  [key: string]: string | number | boolean | string[]
+}) => {
+  return Object.entries(allowedFilterParams(routeParams)).map(([key, value]) => {
+    const paramName = QueryParamsToFilterValueMapping[key]
+    let paramValue = value
+    if (paramValue === "true" || paramValue === "false") {
+      paramValue = paramValue === "true" ? true : false
+    }
```

Comment:
Maybe this is easier to read 🤷 

```suggestion

    let paramValue = value === true || value === "true"
```

Reply:
Ah, ok. What would happen if it is "foobar"?

Reply:
nothing should happen - `paramValue` should have the value of `value`